src/test/java/seedu/address/testutil/TypicalLesson.java

```diff
@@ -0,0 +1,29 @@
+package seedu.address.testutil;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import seedu.address.model.lesson.Day;
+import seedu.address.model.lesson.Lesson;
+
+/**
+ * A utility class containing a list of {@code Lesson} objects to be used in tests.
+ */
+public class TypicalLesson {
+
+    public static final Lesson WED_2_TO_4 = new LessonBuilder().withStartTime("14:00").withEndTime("16:00")
```

Comment:
Perhaps you can consider renaming `WED_2_TO_4` to `LESSON_WED_2_TO_4`, so as to avoid confusing this constant as a time object.